### PR TITLE
Upgrade dep by regular upgrade

### PR DESCRIPTION
**Pull request by takara9**
This PR is the part of the regular update work.

### CODE REVIEW

**Review by morimoto-cybozu — CHANGES_REQUESTED** [review]
Please clean up your commits before requesting review.

I'm concerned about unmaintained components, e.g., node-exporter.

**Comment by takara9** [comment]
@morimoto-cybozu 
Thank you as always. I really appreciate it.
I did cleanup commit messages and update dependency modules which include node-exporter etc.

File: Makefile.common [hunk 1]
```diff
@@ -12,34 +12,34 @@ STERN_VERSION = 1.27.0
 
 
 ## These should be updated regularly
-ARGOCD_VERSION = 2.10.1
+ARGOCD_VERSION = 2.10.2
 # Follow Argo CD installed kustomize version
 # https://github.com/cybozu/neco-containers/blob/main/argocd/Dockerfile#L10
-KUSTOMIZE_VERSION = 5.2.1
+KUSTOMIZE_VERSION = 5.4.1
```
Comment (morimoto-cybozu) [inline Makefile.common:18]: Argo CD v2.10.2 uses Kustomize v5.2.1.
Argo CD v2.10.7 also uses Kustomize v5.2.1.

```suggestion
KUSTOMIZE_VERSION = 5.2.1
```

File: Makefile.tools [hunk 2]
```diff
@@ -264,7 +264,6 @@ kustomize: $(KUSTOMIZE_DOWNLOAD)
 	$(CP) $(KUSTOMIZE_DLDIR)/LICENSE $(DOCDIR)/$@/LICENSE
 	$(CP) $(KUSTOMIZE_DLDIR)/README.md $(DOCDIR)/$@/README.md
 	unzip -o $(KUSTOMIZE_DLDIR)/windows_amd64/kustomize.zip -d $(KUSTOMIZE_DLDIR)/windows_amd64
-	$(CP) $(KUSTOMIZE_DLDIR)/windows_amd64/kustomize $(WINDOWS_BINDIR)/kustomize.exe
```
Comment (morimoto-cybozu) [inline Makefile.tools:267]: Why?

**Review by morimoto-cybozu — CHANGES_REQUESTED** [review]

File: Makefile.tools (hunk 2 above)
Comment (takara9) [reply Makefile.tools:267]: Exe file name is changed in kustomize 5.4.1.

**Pull request closed without merging**